conf/fub_curta.config

```diff
@@ -1,20 +1,23 @@
 // Config profile metadata
 params {
+    config_profile_contact = 'Wassim Salam (@wassimsalam01)'
     config_profile_name = 'FUB Curta'
-    config_profile_description = 'Freie Universität Berlin HPC cluster (Curta) profile'
-    config_profile_contact = 'hpc@zedat.fu-berlin.de'
+    config_profile_description = 'Freie Universität Berlin HPC (Curta) profile'
     config_profile_url = 'https://www.fu-berlin.de/en/sites/high-performance-computing/index.html'
-    max_memory = 93.GB
-    max_time = 14.d
+
     max_cpus = 32
+    max_memory = 385.GB
+    max_time = 14.d
+    // These parameters exclude the 4 x 772.GB compute nodes on 'main'
```

Comment:
Is there a reason for this?

Reply:
Hi James. Perhaps my comment could have been a bit clearer and is a bit misleading. These 4 nodes could potentially be used by Slurm, however with the `max_memory` being capped at 385.GB, a user will not be able to request more than that for 1 job.

I made it this way as of now for simplicity's sake. Below is a table with some stats about my cluster

| nb of nodes | partition     | cpus | memory (MBs)  |
|------------|-----------|------|-----------|
| 2          | begendiv  | 96   | 154657    |
| 2          | begendiv  | 96   | 386310    |
| 50         | main      | 32   | 192080    |
| 14         | main      | 32   | 385600    |
| 4          | main      | 32   | 772700    |
| 98         | main      | 32   | 95320     |

My thought was that I simply wish for Slurm to allocate the job wherever possible, while giving priority to the `begendiv`, since it is an organisation-exclusive partition.

With the way the the `--partition` argument is currently set up, if I had specified `max_memory=772.GB`, then wouldn't this possibly throw an error if Slurm tried to allocate a job to the `begendiv` partition with more than `386.GB`?

Lastly, I doubt a single job would ever need more than that much memory, hence why I thought it was not worth the trouble for now ^^ if someone encounters this while working with this profile I would be more than happy to give it a rework

Reply:
Ah ok, if the very large partition is limited to a specific org/group, then ti's fair enough as a basic default to cap to just the public ones.

> Lastly, I doubt a single job would ever need more than that much memory

Oh you underestimate bioinformatics ;) I regularly have to use 1TB nodes for metagenomics :grimacing:

Reply:
oh dear x) yeah haven't yet worked on anything of that scale. I'm working with a small organism so perhaps I was biased when making this. I'm still open to improving this profile.

Just to be sure, from your sentence I understand that you understood `begendiv` is the public partition, correct? It is the opposite: `begendiv` is org exclusive and `main` is the public default one. So ideally yes I want to up `max_memory` to 772.GB since these nodes are available to all users, but again I am not sure how Slurm would behave in this situation when given `--partition=begendiv,main`. Would it then automatically always assign it to `main` and ignore `begendiv` (whose biggest node has 386.GB)?

I would love to test this somehow but I'm certain I would get a stern email from my HPC team asking me to stop allocating this much memory just for "testing purposes" ^^

Reply:
Sorry yes, I got it the wrong way around. 

Basically setting your cap should be to whatever is the maximum _all users_ can access. I don't know either for SLURMs intepretation of `begendiv,main`, but you could do a condition in the profile.

Reply:
Okay I see :) thank you for your help James. Then I'll probably make one more PR to update this profile soon enough.